supervisor/install/install_windows.go

```diff
@@ -37,32 +37,35 @@ func Install(options Options) error {
 	}
 	defer m.Disconnect()
 
+	log.Println("Cleaning up previous installation")
+	uninstallService(m, serviceName)
+
 	config := mgr.Config{
 		ServiceType:  windows.SERVICE_WIN32_OWN_PROCESS,
 		StartType:    mgr.StartAutomatic,
 		ErrorControl: mgr.ErrorNormal,
 		DisplayName:  "MystSupervisor Service",
 		Description:  "Mysterium Network dApp supervisor service is responsible for managing network configurations",
```

Comment:
On windows, this is actually visible to the users, hence I would improve the wording.

```suggestion
		DisplayName:  "MysteriumVPN Supervisor",
		Description:  "Handles network configuration for MysteriumVPN application.",
```

Reply:
Improved. 
Dont we want to rename `const serviceName = "MystSupervisor"` also (mac, windows)?

Reply:
Service name now is `MysteriumVPNSupervisor`